gpawsolve.py

```diff
@@ -41,7 +42,9 @@
  | ------ | ------------------- | ----------- | -------------- | --- | ---- | ---------------- | ------- |
  |   PW   | Yes                 | Yes         | Yes            | Yes | Yes  | Yes              | Yes     |
  | PW-G0W0| Yes                 | Yes         | No             | No  | Yes  | No               | No      |
+ | PW-EXX*| Yes (with PBE)      | No          | No             | No  | No   | No               | No      |
  |  LCAO  | No                  | No          | No             | Yes | Yes  | Yes              | No      |
+ *: Just some ground state energy calculations for PBE0 and HSE06.
```

Comment:
EXX calculations are so premature in GPAW for an academic work. However, as you pointed, it is a starting point for future.